mayastor/src/bin/mayastor.rs

```diff
@@ -42,12 +42,33 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     let free_pages: u32 = sysfs::parse_value(&hugepage_path, "free_hugepages")?;
     let nr_pages: u32 = sysfs::parse_value(&hugepage_path, "nr_hugepages")?;
     let uring_supported = uring::kernel_support();
+    let nvme_core_path = Path::new("/sys/module/nvme_core/parameters");
+    let nvme_mp: String =
+        match sysfs::parse_value::<String>(&nvme_core_path, "multipath") {
+            Ok(s) => match s.as_str() {
+                "Y" => "yes".to_string(),
+                "N" => "disabled".to_string(),
+                u => format!("unknown value {}", u),
+            },
+            Err(e) => {
+                if e.kind() == std::io::ErrorKind::NotFound {
+                    if nvme_core_path.exists() {
+                        "not built".to_string()
+                    } else {
+                        "nvme not loaded".to_string()
+                    }
+                } else {
+                    format!("unknown error: {}", e)
+                }
+            }
+        };
 
     info!("Starting Mayastor ..");
     info!(
         "kernel io_uring support: {}",
         if uring_supported { "yes" } else { "no" }
     );
+    info!("kernel nvme multipath support: {}", nvme_mp);
```

Comment:
I'm not sure about this message, because it can be confusing. Storage node (mayastor) has nothing to do with multipath support. Storage node always supports multipathing because it is our code and spdk. What matters is if the multipathing is supported on the nodes where the application is running and that can be a mayastor node but also entirely different node when storage nodes do just the storage and don't run any applications. In this sense the message is confusing and should be rather removed from here and printed in our mayastor-csi plugin which is the right place for this and would be really useful there. Not sure what others think.

Reply:
I added it because it probably would have helped to debug the CI failures we had but I agree that it only helps if the application and mayastor are running on the same node. @tiago seemed to approve of it. If it helps I can add the word "initiator" after "nvme" to minimise confusion.

Reply:
I've added "initiator" to the message. I think it can help if all nodes in the k8s cluster use the same kernel and kernel configuration. In any case, it's a separate commit so can be easily reverted in future, if necessary.